collect_app/src/main/java/org/odk/collect/android/adapters/AboutListAdapter.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2018 Shobhit Agarwal
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.odk.collect.android.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.odk.collect.android.R;
+
+public class AboutListAdapter extends RecyclerView.Adapter<AboutListAdapter.ViewHolder> {
+
+    private final Context context;
+    private final int[][] items;
```

Comment:
Given that the items aren't dynamic, I think it's somewhat clearer to specify all the contents in the layout. Any particular reason you chose to do it this way?

Reply:
I realized that `AboutPreferenceActivity` has nothing to do with `Shared Preferences` and is wrongly placed in that package. So treating it as a normal activity seems reasonable to me.
Therefore, not using `PreferenceScreen` in the layout.


> Given that the items aren't dynamic, I think it's somewhat clearer to specify all the contents in the layout. Any particular reason you chose to do it this way?

Each item in the row has the following structure:
```
<RelativeLayout>
    <ImageView />
    <Textview />
    <TextView />
</RelativeLayout>
```

So putting everything in a single layout will be copying/pasting the same code 4 times with only different image and text values. So according to me, that code is less flexible and looks more complex. If anyone has to change the appearance of the items in the future then he/she will have to take care if the same changes have been made to the rest as well.

But if we use this approach (adapters and recycler view) then it becomes very easy as for modifying the appearance only the template needs to be modified. Also, adding/removing items can be done without much change.

Reply:
Fair enough. I go back and forth in cases like this.